### PR TITLE
chore: playwright test screenshots components and update React 18

### PR DESCRIPTION
**Pull request by NasgulNexus**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,13 +29,20 @@
...
+    "playwright:docker": "./scripts/playwright-docker.sh 'npm run playwright'",
+    "playwright:docker:update": "./scripts/playwright-docker.sh 'npm run playwright:update'",
+    "playwright:docker:clear-cache": "./scripts/playwright-docker.sh clear-cache"
   },
   "dependencies": {
     "react-transition-group": "^4.4.1",
     "tslib": "^2.4.0"
   },
   "devDependencies": {
+    "@andylacko/vite-svg-react-loader": "^3.0.16",
```
Comment (Lunory) [inline package.json:45]: "This plugin is deprecated" 👀
https://www.npmjs.com/package/@andylacko/vite-svg-react-loader#this-plugin-is-deprecated

File: src/components/ActionBar/__snapshots__/ActionBar.visual.test.tsx-snapshots/ActionBar-render-story-SingleSection-dark-chromium-linux.png
Comment (Lunory) [inline src/components/ActionBar/__snapshots__/ActionBar.visual.test.tsx-snapshots/ActionBar-render-story-SingleSection-dark-chromium-linux.png:1]: Component `ActionBar` doesn't fit on this screenshot

File: src/components/AsideHeader/__snapshots__/AsideHeader.visual.test.tsx-snapshots/AsideHeader-render-story-HeaderAlert-dark-chromium-linux.png
Comment (Lunory) [inline src/components/AsideHeader/__snapshots__/AsideHeader.visual.test.tsx-snapshots/AsideHeader-render-story-HeaderAlert-dark-chromium-linux.png:1]: There is strange effect on top `AsideHeader`

File: src/components/AsideHeader/__snapshots__/AsideHeader.visual.test.tsx-snapshots/AsideHeader-render-story-HeaderAlertCentered-dark-webkit-linux.png
Comment (Lunory) [inline src/components/AsideHeader/__snapshots__/AsideHeader.visual.test.tsx-snapshots/AsideHeader-render-story-HeaderAlertCentered-dark-webkit-linux.png:1]: same thing 🤔

File: src/components/Footer/desktop/__snapshots__/Footer.visual.test.tsx-snapshots/Footer-render-story-ClearView-dark-webkit-linux.png
Comment (Lunory) [inline src/components/Footer/desktop/__snapshots__/Footer.visual.test.tsx-snapshots/Footer-render-story-ClearView-dark-webkit-linux.png:1]: 👀

File: src/components/Footer/mobile/__snapshots__/MobileFooter.visual.test.tsx-snapshots/MobileFooter-render-story-ClearView-dark-chromium-linux.png
Comment (Lunory) [inline src/components/Footer/mobile/__snapshots__/MobileFooter.visual.test.tsx-snapshots/MobileFooter-render-story-ClearView-dark-chromium-linux.png:1]: 👀

File: src/components/HotkeysPanel/__snapshots__/HotkeysPanel.visual.test.tsx-snapshots/HotkeysPanel-render-story-Showcase-dark-chromium-linux.png
Comment (Lunory) [inline src/components/HotkeysPanel/__snapshots__/HotkeysPanel.visual.test.tsx-snapshots/HotkeysPanel-render-story-Showcase-dark-chromium-linux.png:1]: 👀

**Review by Lunory — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (NasgulNexus) [reply package.json]: changed to another plugin vite-plugin-svgr

**Review by Lunory — APPROVED** [review]

**Review by DarkGenius — APPROVED** [review]

**Pull request merged**